koku/masu/database/trino_sql/reporting_awscostentrylineitem_summary_by_ec2_compute_p.sql

```diff
@@ -52,13 +74,13 @@ SELECT uuid() as uuid,
     memory,
     cast(
         map_filter(
-            cast(json_parse(tags) as map(varchar, varchar)),
+            cast(json_parse(tags[1]) as map(varchar, varchar)),
             (k,v) -> contains(pek.keys, k)
         ) as json
-     ) as tags,
-    json_parse(costcategory) as cost_category,
+    ) as tags,
+    cast(json_parse(cost_category[1]) as json) as cost_category,
```

Comment:
Similar to tags, I think we all the cost category values
```suggestion
    cast(json_parse(cost_category) as json) as cost_category,
```